### DISCUSSION TITLE
Use fixtures to create a nice demo database (django)

### DISCUSSION THREAD

**Discussion by guettli** [pytest-dev/pytest#9178] [General]
I am looking for a way to fill an empty database with some data, so that
I have nice demo database

I already use pytest fixtures for testing my django project.

Is there a way to re-use the fixtures to create a persistent demo database?

**Comment by RonnyPfannschmidt** [comment]
Im tasked to create a helper for manual tests which should fit the bill, a release may happen later today (context is manual tests and using pytest to create the working setup around it)

Until then, just create a test to make the db, and pause it

Depending on the setup you might want to use sigterm to destroy the pytest process without destruction of the test database

We ought to create better tools for this

**Comment by nicoddemus** [comment]
>  a release may happen later toda

A pytest release?

**Reply by RonnyPfannschmidt** [reply]
@nicoddemus tools for manual tests, where one might want to run the setup, then have the testrun pause to demo/ploay,  then let the testrun cleanup down the system again, currently the wait and exit later part is not implemented tho,
